src/scenes/AcceleratorRouter/Deliverables/DeliverableView.tsx

```diff
@@ -163,15 +179,25 @@ const DeliverableView = () => {
     [callToAction, optionsMenu]
   );
 
-  const crumbs: Crumb[] = useMemo(
-    () => [
-      {
-        name: activeLocale ? strings.DELIVERABLES : '',
-        to: APP_PATHS.ACCELERATOR_DELIVERABLES,
-      },
-    ],
-    [activeLocale]
-  );
+  const crumbs: Crumb[] = useMemo(() => {
+    if (!activeLocale) {
+      return [];
+    }
+
+    return source?.startsWith('/accelerator/projects/')
+      ? [
+          {
```

Comment:
I'm wondering if there's a better way we can do this in case the path ever changes, but I'm not thinking of anything right now 🤔

Reply:
Yeah I didn't love this either... Just thought of a simple method to make this more resilient, will fix.